src/cdk/overlay/overlay-directives.ts

```diff
@@ -256,7 +256,7 @@ export class ConnectedOverlayDirective implements OnDestroy, OnChanges {
   }
 
   ngOnChanges(changes: SimpleChanges) {
-    if (changes['open'] || changes['_deprecatedOpen']) {
+    if (changes.open || changes._deprecatedOpen) {
```

Comment:
We should stick with the string access here; Closure Compiler may attempt to rename these otherwise